README.md

```diff
@@ -251,6 +251,53 @@ Join our [Google Group](https://groups.google.com/forum/#!forum/medic-developers
 
 You can contribute by messaging our [Google Group](https://groups.google.com/forum/#!forum/medic-developers), raising bugs or improvements in the medic-webapp Github repo, or submitting pull requests.
 
+## Vulnerability Disclosure Policy
+
+We take the security of our systems seriously, and we value the security community. The disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+### Guidelines
+
+We require that all researchers:
+
+- Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing;
+- Perform research only within the scope set out below;
+- Use the identified communication channels to report vulnerability information to us; and
+- Keep information about any vulnerabilities you've discovered confidential between yourself and Medic Mobile until all production systems have been patched.
+
+If you follow these guidelines when reporting an issue to us, we commit to:
+
+- Not pursue or support any legal action related to your research;
+- Work with you to understand and resolve the issue quickly (including an initial confirmation of your report within 72 hours of submission); 
+- Recognize your contribution on our Security Researcher Hall of Fame, if you are the first to report the issue and we make a code or configuration change based on the issue.
+
+### Scope
+
+- https://alpha.dev.medicmobile.org
```

Comment:
I think this is fine for now, but we could eventually create a special container (even isolated if we were paranoid) once we're all the way over to Docker – just to keep it out of the way of our testing.